src/Core/src/CommandMapper.cs

```diff
@@ -43,25 +48,38 @@ private protected virtual void InvokeCore(string key, IElementHandler viewHandle
 				return null;
 		}
 
-		internal void Invoke(IElementHandler viewHandler, IElement? virtualView, string property, object? args)
+		public void Invoke(IElementHandler viewHandler, IElement? virtualView, string property, object? args)
 		{
 			if (virtualView == null)
 				return;
 
 			InvokeCore(property, viewHandler, virtualView, args);
 		}
 
-		public CommandMapper? Chained
+		public ICommandMapper? Chained
```

Comment:
This is the breaking change. The API is the same - or rather before there was just `GetCommand`, but now there are 2 methods.

The break is the type change of the property. It is now the base interface.

Reply:
Let me see if I can find a way around this...

Reply:
Fixed! No reason to make things use interfaces in this PR.